examples/cull-idle/Cull_service_UI/Readme.md

```diff
@@ -0,0 +1,31 @@
+<h2><strong>Cull Service UI Monitoring</strong></h2>
```

Comment:
The filename suggests this is a markdown file, but this is HTML. Did you commit the wrong file?